fdbserver/RestoreApplier.actor.cpp

```diff
@@ -487,4 +512,4 @@ Value applyAtomicOp(Optional<StringRef> existingValue, Value value, MutationRef:
 		ASSERT(false);
 	}
 	return Value();
-}
+}
```

Comment:
Maybe add a new line?

Reply:
Hmm interesting. I didn't remember I did anything with this line. Maybe a space was added or deleted? (But I cannot see that change either from the diff.)
Maybe I deleted an empty line at the end of this file.